src/components/BrandConfigProvider/styles/StylesProvider.tsx

```diff
@@ -1,26 +1,25 @@
 import * as React from 'react';
-import { merge } from 'lodash';
+import merge from 'lodash/merge';
 import {
   ComponentStyles,
   RecursivePartial,
   StylesBuilder,
   NamedStyles,
   BrandConfigProviderStyles,
 } from './types';
-import { Theme } from '../theme/Theme';
-import { useTheme } from '../../../hooks/useTheme';
+import { useTheme } from '../theme/ThemeProvider';
 
 export const StylesContext = React.createContext({
-  componentStyles: {} as RecursivePartial<ComponentStyles>,
+  styles: {} as RecursivePartial<ComponentStyles>,
```

Comment:
I feel like eventually we might want to allow styles to be an object or a function that accepts the theme as an argument. This would allow a consumer to override some parts of the theme, and then construct the component style from the built theme. That way if they want to change the background and foreground colors of a component it would be easy to switch from primary/onPrimary (default) to secondary/onSecondary (custom) if they wanted and wouldn't require that they set that up ahead of time to override that color via the `StylesProvider` prop.